packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.13.1"
+  changes:
+    - description: Update documentation with additional context for new users. 
```

ci is failing because there's an extra space at the end of this line

```suggestion
    - description: Update documentation with additional context for new users.
```

---

The ci error isn't the most helpful, but in my experience when the error is related to the changelog file, there's almost always an extra space at the end of the description.